packages/datetime/src/_datepicker.scss

```diff
@@ -202,44 +201,43 @@ $header-margin: ($header-height - $pt-input-height) / 2 !default;
     display: table-cell;
     border-radius: $pt-border-radius;
     cursor: pointer;
-    width: $cell-size;
-    height: $cell-size;
+    width: $datepicker-day-size;
+    height: $datepicker-day-size;
     vertical-align: middle;
     text-align: center;
     line-height: 1;
 
-    &:focus {
-      outline: 0;
+    // spelling out full name so these are equal specificity to pseudo-classes (.DayPicker-Day:hover)
+    &.DayPicker-Day--outside {
+      color: $pt-text-color-disabled;
     }
 
-    &:not(.DayPicker-Day--disabled):not(.DayPicker-Day--outside):hover,
-    &:not(.DayPicker-Day--disabled):not(.DayPicker-Day--outside):focus {
-      background: $light-gray1;
+    &:hover,
+    &:focus {
+      background: $datepicker-day-background-color-hover;
       color: $pt-text-color;
     }
-  }
 
-  .DayPicker-Day--disabled {
-    cursor: not-allowed;
-    color: $pt-text-color-disabled;
-  }
-
-  .DayPicker-Day--outside {
-    cursor: default;
-  }
+    &:active {
+      background: $datepicker-day-background-color-active;
+    }
 
-  .DayPicker-Day--selected:not(.DayPicker-Day--outside) {
-    border-radius: $pt-border-radius;
-  }
+    &.DayPicker-Day--selected {
+      border-radius: $pt-border-radius;
+      background-color: $blue3;
+      color: $white;
 
-  .DayPicker-Day--selected:not(.DayPicker-Day--outside),
-  .DayPicker-Day--selected:not(.DayPicker-Day--outside):focus {
-    background-color: $blue3;
-    color: $white;
+      &:hover {
+        background-color: $blue2;
```

Comment:
should focus  have `$blue2` too (since we do that for the non selected)? if so, same for the dark colors below